metaci/users/management/commands/tests/test_packaging_org_users.py

```diff
@@ -66,8 +66,8 @@ def test_run_command(self, salesforce_client, refresh_oauth):
         OrgFactory(name="packaging", repo__name="RepoD")
         salesforce_client.query = fake_query
         c = Command()
-        OrgConfigModule = sys.modules["cumulusci.core.config.OrgConfig"]
-        with mock.patch.object(OrgConfigModule, "SKIP_REFRESH", True):
+        print(sys.modules.keys())
+        with mock.patch("cumulusci.core.config.org_config.SKIP_REFRESH", True):
```

Comment:
@prescod shouldn't this patch be unnecessary? SKIP_REFRESH is only used inside `refresh_oauth_token`, which is patched in a decorator above.